opam.sh

```diff
@@ -4,10 +4,9 @@ set -e
 
 rm -rf opam
 
-mkdir opam
-mkdir opam/hacl-star-raw
+mkdir -p opam/hacl-star-raw/build
```

Comment:
Why do we need the `build` now?

Reply:
Do you mean that you want to create the folder as late as possible to make the code more understandable?